### ISSUE TITLE
Service Error

### ISSUE THREAD

**Issue by James-W2** [web-scrobbler/web-scrobbler#2511]
**Describe the bug**
When playing from Amazon Music, this evening sporadically quit scrobbling to Last.FM and extension icon turned amber.  Error message stated: "Service error. Looks like something went wrong while scrobbling the music. Please open the options page and try to reauthenticate with services."

However, on the options page I cannot find anything that reauthenticates with services.  I understand this happens only when first install the extension.  Where do I go from here?

**How to reproduce**
Basically just playing music in Amazon Music website.

**Expected behavior**
The song/musical piece will scrobble in Last.fm

**Environment (please complete the following information)**
-   OS:  Windows 7 Pro
-   Browser: (Chrome)
-   Extension version: (v2.34.0)

**Additional context**

**Comment by alexesprit** [comment]
![image](https://user-images.githubusercontent.com/1119267/89411200-b2b61900-d72d-11ea-881a-a3495d9c6ce8.png)
